apps/modeler/include/pcl/apps/modeler/main_window.h

```diff
@@ -128,7 +128,7 @@ namespace pcl
         friend class AbstractItem;
 
         MainWindow();
-        MainWindow(const MainWindow &) {} // copy ctor hidden
+        MainWindow(const MainWindow &) = default;
```

Comment:
Line can be deleted, because it will be automatically generated. See [here](https://docs.microsoft.com/en-us/cpp/cpp/explicitly-defaulted-and-deleted-functions?view=vs-2019)